### PR TITLE
Add repeated characters question

### PR DESCRIPTION
**Pull request by apicardgoog**
This question is intended to be added to the textbook.

### CODE REVIEW

File: assets/questions/findMostCommonRepeatedCharacter.js [hunk 1]
```diff
@@ -0,0 +1,276 @@
...
+`
+  },
+  tasks: [{
+    id: 'findMostCommonRepeatedCharacter',
+    instructions: [{
+      content: [
+        'In this problem, we\'d like you to write a function that determines ',
+        'the most common repeated character in an ASCII string (ignoring ',
+        'spaces and punctuation). ',
+        'Note that uppercase and lowercase letter should be treated as the ',
```
[the remark this answers is not in the document]
Comment (apicardgoog) [reply assets/questions/findMostCommonRepeatedCharacter.js]: Done

[the remark this answers is not in the document]
Comment (apicardgoog) [reply assets/questions/findMostCommonRepeatedCharacter.js]: Done

File: assets/questions/findMostCommonRepeatedCharacter.js [hunk 2]
```diff
@@ -0,0 +1,276 @@
...
+      if most_common:
+        return most_common[0][0]
+`
+  },
+  tasks: [{
+    id: 'findMostCommonRepeatedCharacter',
+    instructions: [{
+      content: [
+        'In this problem, we\'d like you to write a function that determines ',
+        'the most common repeated character in an ASCII string (ignoring ',
```
[the remark this answers is not in the document]
Comment (apicardgoog) [reply assets/questions/findMostCommonRepeatedCharacter.js]: I think there are two questions here.
1. If this is too hard to Fuzzy, we could use the simpler problem but that doesn't require this work because it's already here:
https://github.com/google/tie/blob/master/assets/questions/findMostCommonCharacter.js

2. Whether "aaa" counts as 1 or 2 repeats: I don't care, we can call it out explicitly either way. Counting it as 2 saves one line of code in the solution.

This also leads to a meta question: how does TIE deal with clarifying questions. In the context of interviews, I intentionally leave questions underspecified to test the interviewee's thinking about the problem and clarify later. In this case, we're not dealing with an interview so should I just call out all the edge cases and how they should be handled? It could be an interesting feature in the future to allow the user to specify edge case inputs before they start coding and to have TIE provide the expected answers for those edge cases.

File: assets/questions/findMostCommonRepeatedCharacter.js [hunk 3]
```diff
@@ -0,0 +1,276 @@
...
+        'spaces and punctuation). ',
+        'Note that uppercase and lowercase letter should be treated as the ',
+        'same letter. ',
+        'You may assume that the input string contains only one most common ',
```
[the remark this answers is not in the document]
Comment (apicardgoog) [reply assets/questions/findMostCommonRepeatedCharacter.js]: Clarified the wording.
